src/main/java/it/gov/pagopa/pu/pagopapayments/service/synchronouspayments/SynchronousPaymentService.java

```diff
@@ -86,8 +93,22 @@ public long retrieveNotificationFeeCents(Long organizationId, String nav, String
         return 0;
       }
     } else {
-      //TODO - P4ADEV-2694 if SENDApiKey doesn't exists call external third part API to retrieve notificationFee
-      return 0;
+      try{
+        List<DebtPositionDTO> debtPositionDTOList = debtPositionService.getDebtPositionsByOrganizationIdAndIuv(organizationId,
+          Utilities.nav2Iuv(nav), ORDINARY_DEBT_POSITION_ORIGINS, accessToken);
+        DebtPositionTypeOrg debtPositionTypeOrg = debtPositionService.getDebtPositionTypeOrgById(debtPositionDTOList.getFirst().getDebtPositionTypeOrgId(), accessToken);
```

Comment:
instead of performing this search using 2 API/queries (resolving PII not used)
let's istead add a query on DebtPositionTypeOrgRepository to search using orgId, nav and origin